pxtlib/docsrender.ts

```diff
@@ -418,13 +418,27 @@ namespace pxt.docs {
 
     export function setupRenderer(renderer: marked.Renderer) {
         renderer.image = function (href: string, title: string, text: string) {
-            let out = '<img class="ui image" src="' + href + '" alt="' + text + '"';
-            if (title) {
-                out += ' title="' + title + '"';
+
+            if (href.startsWith("https://www.youtube.com/embed/")){
```

Comment:
Instead of checking for the entire URL here, can we do something like following in the markdown
youtube: <substring of the video>?
Construct the URL based on that

Reply:
updated to construct url